Source/NSHTTPCookie.m

```diff
@@ -59,6 +59,7 @@ after this (See GSRangeOfCookie).
 NSString * const NSHTTPCookieSecure = @"Secure";
 NSString * const NSHTTPCookieValue = @"Value";
 NSString * const NSHTTPCookieVersion = @"Version";
+static NSString * const NSHTTPCookieHTTPOnly = @"HTTPOnly";
```

Comment:
I would not use a variable name starting with NS for a static variable. Apple uses this name space and we might get a clash later on.